dev-libs/opencryptoki/opencryptoki-3.23.0.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic
+
+DESCRIPTION="PKCS#11 provider cryptographic hardware"
+HOMEPAGE="https://github.com/opencryptoki/opencryptoki"
+SRC_URI="https://github.com/opencryptoki/opencryptoki/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P}"
+
+# Upstream is looking into relicensing it into CPL-1.0 entirely; the CCA
+# token sources are under CPL-1.0 already.
+LICENSE="CPL-0.5"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~s390 ~x86"
+IUSE="debug +tpm"
+
+DEPEND="
+	net-nds/openldap
+	tpm? ( app-crypt/trousers )
+	>=dev-libs/openssl-1.1.0:0=
+"
+RDEPEND="
+	${DEPEND}
+	acct-group/pkcs11
+"
+
+DOCS=(
+	README AUTHORS FAQ TODO
+	doc/openCryptoki-HOWTO.pdf
+)
+
+# tests right now basically don't exist; the only available thing would
+# test against an installed copy and would kill a running pcscd, all
+# things that we're not interested to.
+RESTRICT=test
+
+src_prepare() {
+	default
+	eautoreconf
+}
```

Comment:
Is the **eautoreconf** necessary? Doesn't the tarball come with pre-generated config file or is there a need to regenerate it?

Also:
```
 * QA Notice: command not found:
 * 
 *      ./configure: line 18468: AX_PROG_CC_FOR_BUILD: command not found
```